client/extensions/zoninator/components/settings/zone/post-card.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+
+class PostCard extends Component {
+
+	handleRemove = () => {
+		this.props.remove( this.props.cardIdx );
+	}
+
+	handleMouseDown = ( event ) => {
+		event.stopPropagation();
+		event.preventDefault();
+	}
```

Comment:
Why do we need this?

Reply:
Otherwise clicking on a button still triggers drag&drop behaviour.